src/lib/services/map/providers/google/markerHandle.ts

```
@@ -0,0 +1,57 @@
+            gmpClickable: options.clickable !== false,
+            zIndex: options.zIndex ?? 0,
+        });
+    }
+
+    setPosition(latLng: LatLngLiteral): void {
+        this.marker.position = latLng;
+    }
+
+    getPosition(): LatLngLiteral {
+        const pos = this.marker.position;
+        if (!pos) return {lat: 0, lng: 0};
```

ESLint

[error] Expected { after 'if' condition.

(curly)